server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -267,17 +270,27 @@ boolean isLeader() {
             }
 
             void submitValue(final long value) {
-                masterService.submitStateUpdateTask("new value [" + value + "]", new ClusterStateUpdateTask() {
-                    @Override
-                    public ClusterState execute(ClusterState currentState) {
-                        return setValue(currentState, value);
-                    }
+                onNode(localNode, new Runnable() {
+                        @Override
+                        public void run() {
+                            masterService.submitStateUpdateTask("new value [" + value + "]", new ClusterStateUpdateTask() {
+                                @Override
+                                public ClusterState execute(ClusterState currentState) {
+                                    return setValue(currentState, value);
+                                }
+
+                                @Override
+                                public void onFailure(String source, Exception e) {
+                                    logger.debug(() -> new ParameterizedMessage("failed to publish: [{}]", source), e);
+                                }
+                            });
+                        }
 
                     @Override
-                    public void onFailure(String source, Exception e) {
-                        logger.debug(() -> new ParameterizedMessage("failed to publish: [{}]", source), e);
+                    public String toString() {
+                        return "submitStateUpdateTask: new value [" + value + "]";
```

Comment:
why do we need this toString() here? We run it right away and don't log it anywhere.

Reply:
Left over from earlier when I was enqueueing it, good point.